S/Spot_julia/build_tarballs.jl

```diff
@@ -19,7 +19,8 @@ delete!(Pkg.Types.get_last_stdlibs(v"1.6.3"), uuid)
 
 # Bash recipe for building across all platforms
 script = raw"""
-cd $WORKSPACE/srcdir/spot-2.9.7/
+export MACOSX_DEPLOYMENT_TARGET=10.15
```

Comment:
Lying about the macOS version you're targeting is not quite enough to achieve anything.  You want to take inspiration from something like https://github.com/JuliaPackaging/Yggdrasil/blob/63d5b20f7dbedd7b7ac4de71b190dcfb4b8353c8/B/BOLT/build_tarballs.jl#L146-L149 https://github.com/JuliaPackaging/Yggdrasil/blob/63d5b20f7dbedd7b7ac4de71b190dcfb4b8353c8/B/BOLT/build_tarballs.jl#L14-L24